CHANGELOG.md

```diff
@@ -28,6 +28,9 @@ All notable changes for each version of this project will be documented in this
     - The `isFirstPageDisabled` and `isLastPageDisabled` have been deprecated in favor of the identical `isFirstPage` and `isLastPage` getter.
 - `IgxOverlayService`
     - The `attach` method overload accepting `Type` and `OverlaySettings` now accepts `OverlayCreateSettings` as second parameter. This interface extends `OverlaySettings` with an additional `injector` property used as `ElementInjector` when creating the dynamic component.
+- `IgxCombo`:
+    - **Breaking Change** The `filterable` property is renamed to `disableFiltering`.
```

Comment:
Um, the main breaking is the `filterable` in the `filteringOptions` being removed?
Might not be an easy way to migrate that - depending on usage, should we consider deprecating it instead? ( I know 😑)

IIRC `filterable` was marked as deprecated, so it should have no active usage (hopefully), though that's also breaking, sure.

Reply:
Also, also this should be for 18.2 now?